### PR TITLE
Clean up OLED last character printing

### PR DESCRIPTION
**Pull request by josephscampo**
## Description
On the corne OLED, there is a line which prints the last keystroke logged. As not all keystroke strings are equal in length, if keystroke N prints a shorter line than keystroke N-1, then the printed line on the OLED screen will have a few nonsensical characters at the end. My change prints a newline at the end of the keystroke log, so that any stale characters are cleared.

## Types of Changes

- [ ] Core
- [X] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/crkbd.c [hunk 1]
```diff
@@ -113,6 +113,8 @@ static void oled_render_keylog(void) {
     oled_write(depad_str(last_keycode_str, ' '), false);
     oled_write_P(PSTR(":"), false);
     oled_write_char(key_name, false);
+    oled_write_ln("",false);
```
Comment (drashna) [inline keyboards/crkbd/crkbd.c:116]:
```suggestion
    oled_advance_page(true);
```

**Comment by josephscampo** [comment]
switched to advance page, thanks for the tip @drashna. tested on my kbd 👍

**Review by drashna — DISMISSED** [review]

**Comment by fauxpark** [comment]
Please remove your personal keymaps from this PR.

**Comment by drashna** [comment]
This is also addressed in the rev4.x PR, and this contains personal keymaps.  I'm gonna just close it.

**Pull request closed without merging**